spring-cloud-dataflow-admin-starter/pom.xml

```diff
@@ -87,6 +87,11 @@
 			<scope>test</scope>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-task-core</artifactId>
+			<version>1.0.0.BUILD-SNAPSHOT</version>
```

Comment:
Should be versioned from properties(from parent pom). `1.0.0.BUILD-SNAPSHOT` only for artifacts which are part of a same maven reactor structure.